x-pack/plugins/security/public/security_checkup/security_checkup_service.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  SecurityOSSPluginSetup,
+  SecurityOSSPluginStart,
+} from '../../../../../src/plugins/security_oss/public';
+import { insecureClusterAlertTitle, insecureClusterAlertText } from './components';
+
+interface SetupDeps {
+  securityOssSetup: SecurityOSSPluginSetup;
+}
+
+interface StartDeps {
+  securityOssStart: SecurityOSSPluginStart;
+}
+
+export class SecurityCheckupService {
+  private securityOssStart?: SecurityOSSPluginStart;
```

Comment:
Yep - we do expect the plugin to be available, but marking `securityOssStart` as potentially undefined is more correct since it is only set during the `setup` phase, which _should_ always be called, but there's nothing forcing this to happen (unlike a constructor, which can't not be called)